pyanaconda/modules/subscription/initialization.py

```diff
@@ -19,26 +19,47 @@
 #
 
 import os
+import sys
 
 from dasbus.typing import get_variant, Str
+from pyanaconda.core.configuration.anaconda import conf
 
 from pyanaconda.core import util
 from pyanaconda.core.constants import RHSM_SERVICE_TIMEOUT
 from pyanaconda.modules.common.constants.objects import RHSM_CONFIG
-from pyanaconda.modules.common.constants.services import RHSM
+from pyanaconda.modules.common.constants.services import RHSM, PAYLOADS
 from pyanaconda.threading import threadMgr
 from pyanaconda.modules.common.task import Task
 from pyanaconda.modules.common.errors.task import NoResultError
 
 from pyanaconda.anaconda_loggers import get_module_logger
 log = get_module_logger(__name__)
 
+RHSM_SYSTEMD_UNIT_NAME = "rhsm.service"
+
+
+def check_initial_conditions():
+    """Can the Subscription service run?"""
+
+    # Exclude the initial setup.
+    if PAYLOADS.service_name not in conf.anaconda.kickstart_modules:
+        log.debug("The Payloads module is required. Quit.")
```

Comment:
Please add a "subscription:" suffix for these messages, its used in many other places in the module and related GUI parts and it helps to quickly locate all subscription based log messages. 

Also just "Quit." sounds a bit weird to me, maybe it could be more explicit "Not starting the subscription DBus module." or "Subscription DBus module startup aborted." ?